backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -762,4 +952,106 @@ public int checkLicenseTypeInUse(String id) {
         List<License> usingLicenses = licenseRepository.searchByLicenseTypeId(id);
         return usingLicenses.size();
     }
+
+    // Read nodes from input and save
+    public String addNodes(String jsonString, User user) throws SW360Exception {
+        try {
+            com.liferay.portal.kernel.json.JSONObject jsonObject = JSONFactoryUtil.createJSONObject(jsonString);
+            return addNodes(jsonObject, user);
+        }
+        catch (Exception e){
+            //TODO: handle exception
+        }
```

Comment:
may be we can handle the exception here and in the below catch blocks as well.
please also check the indentation here in the catch block, and in below catch block before { .
indention in line 993 after if. line 1031 after for.